split/testing.txt

```diff
@@ -1,18 +1,18 @@
-coq-bits.1.1.0
-coq-qcert.2.2.0
-coq-ceres.0.4.0
-coq-corn.8.16.0
-coq-bytestring.0.9.0
-coq-hammer.1.3.2+8.11
-coq-gaia-stern.1.15
-coq-mathcomp-apery.1.0.1
-coq-tlc.20200328
-coq-iris-heap-lang.3.4.0
-coq-printf.2.0.0
-coq-smtcoq.2.0+8.11
-coq-topology.10.0.1
-coq-haskell.1.0.0
-coq-bbv.1.3
-coq-poltac.0.8.11
-coq-mathcomp-odd-order.1.14.0
-coq-hott.8.11
+coq-bits.1.1.0
+coq-qcert.2.2.0
+coq-ceres.0.4.0
+coq-corn.8.16.0
+coq-bytestring.0.9.0
+coq-hammer.1.3.2+8.11
+coq-gaia-stern.1.15
+coq-mathcomp-apery.1.0.1
+coq-tlc.20200328
+coq-iris-heap-lang.3.4.0
+coq-printf.2.0.0
+coq-smtcoq.2.0+8.11
+coq-topology.10.0.1
+coq-haskell.1.0.0
+coq-bbv.1.3
+coq-poltac.0.8.11
+coq-mathcomp-odd-order.1.14.0
+coq-hott.8.11
```

Comment:
What is happening here?

Reply:
Converting CRLF -> LF (this is a little unimportant fix aside the main purpose of the branch indeed)